ngi_reports/utils/entities.py

```diff
@@ -347,6 +365,50 @@ def populate(self, log, organism_names, **kwargs):
                                         'ApplicationVersion': fc_runp.get('ApplicationVersion')
                                         }
 
+            ## collect info of samples and their library prep / LIMS indexes on the FC (only if -b option is set)
+            if kwargs.get('barcode_from_fc'):
+                log.info('\'barcodes_from_fc\' option was given so index sequences for the report will be taken from the flowcell instead of LIMS')
```

Comment:
You could use double quotes around barcodes_from_fc here instead of `\'` if you want

Reply:
I prefer to have it consistent with the rest of the code, so will leave it as is. But noted! :)